packages/terra-site/package.json

```diff
@@ -92,6 +92,7 @@
     "babel-loader": "^7.1.2",
     "babel-polyfill": "^6.23.0",
     "clean-webpack-plugin": "^0.1.17",
+    "terra-horizontal-divider": "^0.0.0",
```

Comment:
should be "terra-horizontal-divider": "^x.0",

Reply:
`^0.0.0` is ok here,`^0.x` is also ok, but I believe `^x.0` would mean taking any major version.

Reply:
ah you are right with `^x.0`. Good catch.